packages/angular/projects/ui-angular/src/lib/components/amplify-authenticator/amplify-authenticator.component.ts

```diff
@@ -54,6 +49,12 @@ export class AmplifyAuthenticatorComponent implements AfterContentInit {
   /**
    * Class Functions
    */
+  public get context() {
+    const { signOut } = this.stateMachine.services;
+    const user = this.stateMachine.user;
+    return { signOut, user };
+  }
```

Comment:
I used typescript getters to provide context, because 1) it reevaluates context on every request, which will pick up changes in `user` or other non-static contexts and 2) templates can simply write `context` instead of `context()`.

Reply:
In this case, do we want `const signOut = () => send('SIGN_OUT')` vs. calling the service directly?

Otherwise this is the equivalent of `Auth.signOut()` and the Authenticator won't know to respond?

Reply:
OOh, nevermind – this is from #333 using the Angular state machine service, not _XState's_ `.services` property 🤦

Reply:
Right! These are aliased helpers ✨  I'll clarify the naming conventions next sprint.